ci/releases.yml

```diff
@@ -32,6 +32,18 @@ suite codesign deploy staging-suite:
   tags:
     - deploy
 
+# Publish suite desktop to production
+
+suite-desktop publish prod:
+  stage: deploy to production
+  only:
+    refs:
+      - releases
+  script:
+    - cd packages/suite-desktop
+    - yarn publish
```

Comment:
What what is this doing? We are using `suite-desktop github release` for Github deployment.

Reply:
This does nothing. `yarn publish` will fail because the package is private. I guess this is some kind of leftover. Last time this job run was [10 months ago](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines/177205441). I'd bet we can safely remove this.